cs-offerings/scripts/delete/delete_channel-pods.sh

```diff
@@ -18,12 +18,12 @@ echo "Preparing yaml for createchannel pod for deletion"
 sed -e "s/%PEER_ADDRESS%/${PEER_ADDRESS}/g" -e "s/%CHANNEL_NAME%/${CHANNEL_NAME}/g" -e "s/%PEER_MSPID%/${PEER_MSPID}/g" ${KUBECONFIG_FOLDER}/create_channel.yaml.base > ${KUBECONFIG_FOLDER}/create_channel.yaml
 
 echo "Deleting Existing Create Channel Pod"
-if [ "$(kubectl get pods | grep createchannel | wc -l | awk '{print $1}')" != "0" ]; then
+if [ "$(kubectl get pods createchannel | grep createchannel | wc -l | awk '{print $1}')" != "0" ]; then
```

Comment:
I am curios on why the changes in this file, like the one here, are required?

Reply:
I kept it consistent with the version that was committed in the master branch by rddill. I also observed that $(kubectl get pods | grep createchannel wasn't working. I had to use $(kubectl get pods -a | grep createchannel or $(kubectl get pods createchannel for this to work

Reply:
thanks